build/Dockerfile.debian11

```diff
@@ -17,57 +18,45 @@ RUN apt-get update -y && apt-get upgrade -y && apt-get install -y \
     gnupg2 \
     && apt-get autoremove -y
 
-# get node 1.12.4 dependencies from Ubtuntu 16 + sovrin repos
+# get node dependencies from Ubtuntu 16 + sovrin repos
 RUN \
-    add-apt-repository "deb http://archive.ubuntu.com/ubuntu xenial universe" && \
-    add-apt-repository "deb http://archive.ubuntu.com/ubuntu xenial main" && \
-    add-apt-repository "deb http://archive.ubuntu.com/ubuntu xenial-security main" && \
-    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys CE7709D068DB5E88 40976EAF437D05B5 3B4FE6ACC0B21F32 && \
-    add-apt-repository "deb https://repo.sovrin.org/deb xenial stable" 
-    
-
-RUN apt-get update -y && \
-    apt-get install -y --allow-downgrades \
-#    libssl1.0.0 \
-#    libsodium18 \
-#    libindy-crypto=0.4.5 \
-#    python3-indy-crypto=0.4.5 \
-    indy-node=1.12.6 \
-    # node depends on plenum
-    indy-plenum=1.12.6 \
-    #plenum dependencies
-    python3-ujson=1.33-1build1 \
-    python3-prompt-toolkit=0.57-1 \
-    python3-pygments=2.2.0 \
-    python3-rlp=0.5.1 \
-    python3-sha3=0.2.1 \
-    python3-leveldb=0~svn68-2build4 \
-    python3-sortedcontainers=1.5.7 \
-    python3-pip=8.1.1-2 \
-    python3-portalocker=0.5.7 \
+    add-apt-repository "deb http://archive.ubuntu.com/ubuntu xenial universe" \
+    && add-apt-repository "deb http://archive.ubuntu.com/ubuntu xenial main" \
+    && add-apt-repository "deb http://archive.ubuntu.com/ubuntu xenial-security main" \
+    && apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys CE7709D068DB5E88 40976EAF437D05B5 3B4FE6ACC0B21F32 \
+    && add-apt-repository "deb https://repo.sovrin.org/deb xenial stable" \
+    # Hyperledger Artifactory for latest indy-node
+    && apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 9692C00E657DDE61 \
+    && add-apt-repository "deb https://hyperledger.jfrog.io/artifactory/indy focal stable" \
```

Comment:
You are referencing `focal` here, while above you are using `xenial` - is this on purpose?

Reply:
in short: yes ;) 

I am trying to use the latest available package version of the deb packages

Reply:
concretely `libssl1.0.0` is pulled from the old xenial repos. I will add a comment